main.go

```diff
@@ -42,6 +43,8 @@ func init() {
 
 	utilruntime.Must(sspv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(controllers.InitScheme(scheme))
+	utilruntime.Must(secv1.Install(scheme))
+
```

Comment:
These lines are not needed. The types will be added to scheme in `AddWatchTypesToScheme`.

Reply:
Done